### PR TITLE
DOCKER-129 Improve the process of locally built image

### PR DESCRIPTION
**Pull request by rotty3000**
- add a "no_warm_up" flag
- add a "no_test_image" flag

For example:

```bash
]$ ./build_local_image.sh ~/bundles/ dxp snapshot no_warm_up no_test_image
```

Builds the image in less than 30 seconds.

### CODE REVIEW

**Comment by rotty3000** [comment]
About not warming up, since these are unofficial images used for local dev only the warm up only takes about 10-20 seconds more which I don't think is worth the 5 extra build minutes... then again, it's an optional flag, and so if you care don't use it.

File: build_local_image.sh [hunk 1]
```diff
@@ -37,27 +37,31 @@ function check_usage {
 }
 
 function main {
+	check_utils rsync
```
Comment (Akos-Kreutz) [inline build_local_image.sh:40]: Please move this check to the already existing **check_utils** call in the **check_usage** function.

File: templates/bundle/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,11 @@
-FROM --platform=${TARGETPLATFORM} liferay/jdk11:latest
+FROM --platform=${TARGETPLATFORM} liferay/jdk11:latest AS osbase
```
Comment (Akos-Kreutz) [inline templates/bundle/Dockerfile:1]: What is the purpose of moving the apt-get commands into a different stage?

Comment (rotty3000) [reply templates/bundle/Dockerfile:1]: Stages are cache able by docker. if there is no input changes that stage is taken from cache saving huge amount of time. In this case it was 3-4 minutes.

**Comment by gamerson** [comment]
hey @Akos-Kreutz  any update on this PR?  Please let me know how our team can help out.  Being able to build and test images locally is something we need. Appreciate your efforts in advance!

**Comment by gamerson** [comment]
hey @Akos-Kreutz  I rebased and resent this PR here: https://github.com/liferay/liferay-docker/pull/98
Along with another minor fix for macos.

**Comment by Akos-Kreutz** [comment]
Hi @gamerson 
This PR was moved to here, as we only send PR-s to Brians fork.
https://github.com/brianchandotcom/liferay-docker/pull/281

**Pull request closed without merging**